sdk/core/core/inc/az_http_transport.h

```diff
@@ -85,16 +85,19 @@ AZ_NODISCARD az_result
 az_http_request_get_header(_az_http_request const* request, int32_t index, az_pair* out_header);
 
 /**
- * @brief Get parts of an HTTP request. NULL in accepted to ignore getting any parts, for example,
- * call this function like below to get only the http method and ignore getting url and body
- *   az_http_request_get_parts(request, &method, NULL, NULL)`
+ * @brief Get parts of an HTTP request. `NULL` in accepted to ignore getting any parts, for example,
+ * call this function like below to get only the http method and ignore getting url and body.
+ *   `az_http_request_get_parts(request, &method, `NULL`, `NULL`)`
```

Comment:
Since this is a line of code, you don't need back ticks around the `NULL` parameters. Also, I don't know how embedded back ticks renders.
I would just keep this:
`az_http_request_get_parts(request, &method, NULL, NULL)`